app/boards/shields/reviung34/reviung34.keymap

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/rgb.h>
+#include <dt-bindings/zmk/outputs.h>
+
+
+
+#define DEFAULT 0
+#define SYMNUM 1
+#define NAVFN 2
+#define CONFIG 3
+
+&mt {
+    tapping-term-ms = <165>;
+};
+
+/ {    
+    macros {
+            ZMK_MACRO(save,
+              wait-ms = <30>;
+              tap-ms = <40>;
+              bindings =  <&macro_tap &kp ESC &kp COLON &kp W &kp RET>;
+            )
+    };
```

Comment:
This seems like it's not indented enough when comparing it with the rest of the file.